src/ripple/protocol/Feature.h

```diff
@@ -337,6 +337,7 @@ extern uint256 const featureNonFungibleTokensV1;
 extern uint256 const featureExpandedSignerList;
 extern uint256 const fixNFTokenDirV1;
 extern uint256 const fixNFTokenNegOffer;
+extern uint256 const featureNonFungibleTokensV1_1;
```

Comment:
You need to increment `numFeatures`. https://github.com/XRPLF/rippled/blob/83aa1c1818e2854e98eebce040b9dc6c72de640e/src/ripple/protocol/Feature.h#L77